Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG CMAKE_VERSION=3.13.4
 ARG PROTOBUF_VERSION=3.9.0
 
 # Install the basics
-RUN apt-get update && apt-get install -y curl python-software-properties build-essential xz-utils libreadline-dev
+RUN apt-get update && apt-get install -y curl python-software-properties build-essential xz-utils libreadline-dev nano
```

Comment:
nano is not required and we should avoid changing lower Docker layer

Reply:
OK, I just felt awkard that no editor, not even vi is present in the docker image :)